content/javascript/concepts/objects/terms/getOwnPropertyDescriptor/getOwnPropertyDescriptor.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Object.getOwnPropertyDescriptor()' 
+Description: 'Enables access to full information of a property.' 
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Web Development'
+  - 'Computer Science'  
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Objects'
+  - 'Methods'  
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`Object.getOwnPropertyDescriptor()`** method is a built-in object that enables access to full information about a property and returns a property descriptor of the given property if it exists on the object, otherwise it returns `undefined`.
+
+## Syntax
+
+Here's the basic syntax of `Object.getOwnPropertyDescriptor()`:
```

Comment:
Please give a line space before and after a code block.

Reply:
On it